asteroidsyncserviced/platforms/ubuntutouch/ubuntuplatform.h

```diff
@@ -2,6 +2,7 @@
 #define UBUNTUPLATFORM_H
 
 #include <QDBusContext>
+#include <QDBusInterface>
```

Comment:
Why is this here ? same for the new include in ubuntuplatform.cpp

Reply:
The interface uses QDBusInterface, but there is no corresponding #include.  Apparently we assume <QDBusContext> or some other header also includes it.  I'll omit it, but I'm not sure of the rationale.